security/security-core/src/main/java/io/camunda/zeebe/protocol/record/value/PermissionType.java

```diff
@@ -11,6 +11,7 @@ public enum PermissionType {
   CREATE,
   READ,
```

Comment:
[Question] I assuming we have 3 different reads to this case, once the permission to read instance, may differ to user task, as well a generic one. So it is not possible have only READ here.  Correct?

Reply:
Yes, ProcesDef.READ allows you to read the process definitions but not the instance objects and the user tasks.
This was specified in https://docs.google.com/document/d/1ikxw9ARCv-MBwASnj7WhW5uHCndCRk2B35xi-cB2N0M/edit?tab=t.0#heading=h.m2zg2f7pb01i